Calculator/ViewModel/CalculatorOperationsProtocol.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+enum Operation {
+    case Addition
+    case Subtraction
+    case Multiplication
+    case Division
+}
+
+protocol CalculatorOperationsProtocol {
+    func operate(opt1: Double, opt2: Double, operation: Operation) -> Double
+}
+
+extension CalculatorOperationsProtocol {
+
+    func operate(opt1: Double, opt2: Double, operation: Operation) -> Double {
+
+        switch operation {
+        case .Addition:
```

Comment:
I like the idea and it's worth talking about it but in the next PR it all is changed. Could we discuss this implementation there? :)